### PR TITLE
Desktop_Portfolio_Form

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Form

- add validation to form
- use if condition 
-  email in lowercase

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const emailError = document.querySelector('.form-error');
+
+form.addEventListener('submit', (e) => {
+  const firstName = form.name.value;
+  const lastName = form.name.value;
+  const email = form.email.value;
+  const message = form.message.value;
+  if (email !== email.toLowerCase()) {
+    e.preventDefault();
+    emailError.innerHTML = `<p class="error">Email should be written in lowercase letters: ${email}</p>`;
```
Comment (leonardodiasb) [inline form.js]: - [ ] Your validation works and the error message appears on screen but it is not appearing in the correct position. Kindly fix the style of the error message.
 - Mobile view:
 - 
image
 - Desktop view:
 - 
image

File: index.html [hunk 2]
```diff
@@ -219,18 +131,18 @@ <h1> About me</h1>
...
-         <form action="https://formspree.io/f/mdojzzgj" method="post">
+         <form action="https://formspree.io/f/mdojzzgj" method="post" class="form">
              
           <input type="text" class="item item1" name="firstName"  placeholder="First Name" required   maxlength="30">
           <input type="text" class="item item2" name="lastName" placeholder="Last Name" required   maxlength="30">
             <input type="email" class="item" placeholder="Email address" name="email" required >
            
             <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Hello Mariane. Glad to see you |"   maxlength="500" required></textarea>
-           
+            <div class="form-error"></div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your "form-error" container is always appearing on the screen even before the user submits the form. Please, make it invisible and only appear if the form is not valid with its message inside.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @AhmadiGu, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @AhmadiGu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**